wiki/en/Tips-Tricks-More.md

```diff
@@ -102,6 +96,16 @@ Make sure you connect your MIDI device's output port to the Jamulus MIDI in port
 
 *Tip*: When you enable MIDI control in Jamulus, each user's name is prepended with a number, with the leftmost user starting at 0, then 1, etc. With default settings, when some users leave and others join, their left-right arrangement in the GUI may cease to follow a numerical order, making it more difficult to know who each physical fader/knob on your MIDI controller corresponds to. In order to keep the fader strips following a numerical order, go to "View" on the top menu bar and choose "Sort Users by Name".
 
+# For Server admins
+
+## Converting a public Server to a private one on the fly
+
+You can run a public Server long enough for your band to connect, then go private by simply unchecking the 'Make my Server Public' box in the Server GUI. Your band mates will still be connected to the Server until they disconnect. (Thanks to [David Savinkoff](https://github.com/DavidSavinkoff) for this tip!)
+
+## Remote management of recordings
+
+Jamulus user [vdellamea](https://github.com/vdellamea) has written a [web-based remote tool](https://github.com/vdellamea/jamulus-server-remote) for starting and stopping recordings on Linux Servers, allowing you to then download them from your browser. See also [Jamulus Jam Exporter](https://github.com/pljones/jamulus-jamexporter) by [pljones](https://github.com/pljones), which also includes a Server recording recovery script.
```

Comment:
JSON-RPC could/should be added too.

Reply:
Not sure I'm the person to do that though.

Reply:
Just add something like "Jamulus also supports JSON-RPC for controlling client and server. Please have a look at the API-documentation (link to JSON-RPC.md)"

Reply:
For it to be a "tip" it needs to actually solve a problem, or help you do something specific though. Simply linking to some docs isn't what this page is about.

Reply:
Hmm. Probably? It's still somewhat related though.

Reply:
So for example, you can use MIDI with `ctrlmidich` to control fader, pan control and mute and solo buttons. If there's something specific like that we can write a HOWTO for using JSON-RPC, then maybe put that in?